src/App.tsx

```diff
@@ -15,36 +16,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  LENGTH = 'L',
+  ALPHABET = 'A',
+  DEFAULT = '',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  isSorted: string,
```

Comment:
```suggestion
  sortType: SortType,
```
It's not boolean, so it shouldn't start with `is`, and it should have corresponding type - `SortType`